packages/devtools_app/lib/src/tree.dart

```diff
@@ -68,7 +68,6 @@ class _TreeViewState<T extends TreeNode<T>> extends State<TreeView<T>>
     if (items.isEmpty) return const SizedBox();
     return ListView.builder(
       itemCount: items.length,
-      itemExtent: widget.itemExtent,
```

Comment:
instead of removing this, can we remove the default value in the constructor? this would make sure that other consumers of tree view can still pass in item extent if they need

Reply:
Ah yeah. That's much better. Done.